src/chainparams.cpp

```diff
@@ -283,6 +283,9 @@ class CTestNetParams : public CMainParams
         nEnforceNewSporkKey = 1521604800; //!> Sporks signed after Wednesday, March 21, 2018 4:00:00 AM GMT must use the new spork key
         nRejectOldSporkKey = 1522454400; //!> Reject old spork key after Saturday, March 31, 2018 12:00:00 AM GMT
 
+        // Fake Serial Attack
+        nFakeSerialBlockheightEnd = 0;
```

Comment:
This needs to be `-1` to avoid an unnecessary recalculation of non-existent accumulators and zerocoin balances when connecting block 1 in the chain. `ConnectBlock()`'s use of this adds a static `+ 1` to the `nHeight`